internal/generator/fluentd/ingress.go

```diff
@@ -197,8 +197,8 @@ const KubernetesMetadataPlugin string = `
   @type kubernetes_metadata
   kubernetes_url 'https://kubernetes.default.svc'
   annotation_match ["^containerType\.logging\.openshift\.io\/.*$"]
+  allow_orphans false
```

Comment:
@alanconway please confirm you on board with this change.  Essentially this makes it so regress of fetch meta you will minimum get: namespace, podname, container name.  There is no way to retrieve pod meta by the more unique id that is available from orphaning.  This will lead us to resorting to cluster policy to control the "deleted" namespace issue.  I will only be able to fix it otherwise by making some change in the actual plugin, otherwise we will need to hack the tests some how